src/coreclr/jit/block.h

```diff
@@ -1435,10 +1476,10 @@ class BasicBlockIterator
     {
         assert(m_block != nullptr);
         // Check that we haven't been spliced out of the list.
-        assert((m_block->bbNext == nullptr) || (m_block->bbNext->bbPrev == m_block));
-        assert((m_block->bbPrev == nullptr) || (m_block->bbPrev->bbNext == m_block));
+        assert((m_block->IsLast()) || m_block->Next()->PrevIs(m_block));
+        assert((m_block->IsFirst()) || m_block->Prev()->NextIs(m_block));
```

Comment:
With the function, we can remove an unnecessary set of parens
```suggestion
        assert(m_block->IsLast() || m_block->Next()->PrevIs(m_block));
        assert(m_block->IsFirst() || m_block->Prev()->NextIs(m_block));
```

Reply:
Thanks for catching all these nits (sorry, these kinds of PRs seem especially tedious to review).